book/website/collaboration/remote-collab/remote-collab-prosandcons.md

```diff
@@ -1,9 +1,13 @@
 (cl-remote-collab-prosandcons)=
-# Perceived Pros and Cons
+# Pros and Cons
 
 It is needless to say that the same setting can not always work for everyone, which applies to remote collaboration, working, and event participation.
-For many, it may not always feel as effective as working in physical offices, for example, when they live in a small apartment, share their place with a large family, have an inadequate remote-work setting or feel less connected with their colleagues.
-However, for others, remote working may be more enjoyable and productive.
+For many, remote collaboration is the most ideal setting, where as for for many this might leave them feeling isolated.
```

Comment:
two "for"s where only one is needed

Reply:
Good catch, just updated in the main file. Thank you.